File: wej-wyj.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  wej-wyj.py
#  
def name ():
    name = input("Podaj swoje imię: ")
    surname = input("Podaj swoje nazwisko: ")
    
    print("Witaj" + " " + name + " " + surname)

def main(args):
    
    name()
    
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: Cpp/wej-wyj.cpp
/*
 * wej-wyj.cpp
 * 
 */


#include <iostream>
using namespace std;

int main(int argc, char **argv)
{
	char imie[10];
    char nazwisko[10];
    
    cout << "Podaj imię: ";
    cin >> imie;
    cout << "Podaj nazwisko: ";
    cin >> nazwisko;
    
    cout << "Witaj " << imie << " ";
    cout << nazwisko;
    
	return 0;
}

